epub33/rs/index.html

```diff
@@ -341,13 +341,15 @@ <h4>Metadata</h4>
 					<dl class="conformance-list">
 						<dt id="conf-meta-ws">White Space</dt>
 						<dd>
-							<p>Reading Systems MUST trim all leading and trailing <a
-									href="https://www.w3.org/TR/xml/#white">white space</a> [[!XML]] from Dublin Core
-								element values before processing.</p>
-							<p>Unless an individual property explicitly defines a different white space normalization
-								algorithm, Reading Systems MUST trim all leading and trailing <a
-									href="https://www.w3.org/TR/xml/#white">white space</a> [[!XML]] from
-									<code>meta</code> element values before further processing them.</p>
+							<p>Reading Systems MUST <a
+									href="https://infra.spec.whatwg.org/#strip-leading-and-trailing-ascii-whitespace"
+									>strip leading and trailing ASCII whitespace</a> [[!Infra]] from Dublin Core element
+								values before processing.</p>
+							<p>Unless an individual property explicitly defines a different whitespace normalization
+								algorithm, Reading Systems MUST <a
+									href="https://infra.spec.whatwg.org/#strip-leading-and-trailing-ascii-whitespace"
+									>strip leading and trailing ASCII whitespace</a> [[!Infra]] from <code>meta</code>
+								element values before further processing them.</p>
```

Comment:
I was wondering what "element values" meant (when reading the preview), before seeing that this PR defined that term in the core spec. I suggest (see comment above) a more specific term is used (e.g. "metadata value" or "metadata expression value"), and that its usage here points to its definition in the Core spec.